base/array.jl

```diff
@@ -1745,12 +1771,16 @@ julia> insert!(Any[1:6;], 3, "here")
 ```
 """
 function insert!(a::Array{T,1}, i::Integer, item) where T
+    @inline
```

Comment:
```suggestion
    @_propagate_inbounds_meta
```
since now `_insert!` contains `@inbounds`

Reply:
Good catch 👍